editor/svgcanvas.js

```diff
@@ -186,9 +186,12 @@ var selectedElements = [];
 // * element - tag name of the SVG element to create
 // * attr - Object with attributes key-values to assign to the new element
 // * curStyles - Boolean indicating that current style attributes should be applied first
+// * children - Optional array with data objects to be added recursively as children
 //
 // Returns: The new element
 var addSvgElementFromJson = this.addSvgElementFromJson = function(data) {
+	if(typeof(data) == 'string') return svgdoc.createTextNode(data);
```

Comment:
Please follow the convention of putting a space after if and before (

`if (typeof(data) == ...`